apache-maven/src/assembly/maven/lib/ext/hazelcast/README.txt

```diff
@@ -0,0 +1,5 @@
+This directory is intended to contain Hazelcast JARs for Maven Resolver Named Locks using Hazelcast.
```

Comment:
would it be worth it to add a link to redisson/hazelcast github?

Reply:
Sure, make a proposal.